_posts/2023-05-25-10block-old-decoy-selection-bug.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: Post-Mortem of 10-Block-Old Decoy Selection Bug
+summary: Emprical analysis of the high severity vulnerability within the reference wallet. Updating to the latest version is strongly advised.
+tags: [urgent, crypto, community]
+author: jeffro256
+---
+
+## Quick Facts
+
+* Severity: HIGH
+* Affected versions: GUI/CLI wallet versions v0.13.0.2 to v0.18.2.1
+* Impact: Loss of sender anonymity for transactions spending funds exactly 10 blocks old
+* What to do: [Update to v0.18.2.2 and above]({{ site.baseurl }}/downloads)
+
+## Introduction
+
+When the Monero wallet needs to construct a @transaction with a certain input, it also picks decoys inputs from the chain (@ring-signature) based on a certain distribution called a "gamma distribution". The gamma distribution makes picking recent decoys more likely than picking older decoys, mimicking how real users spend their funds. However, in the gamma picker code, there was a off-by-one bug that didn't allow the gamma picker to pick decoys which are exactly 10 blocks old.    
```

Comment:
Links like `@transaction` and `@ring-signature` won't work on blog posts

Reply:
Did something change to prevent this? (i have copied of the use of them in this blog post https://www.getmonero.org/2021/09/20/post-mortem-of-decoy-selection-bugs.html) which i notice also has a `{% t global.lang_tag %}` after the front matter, is this also required?

Reply:
Yes, either remove the links, or add `{% t global.lang_tag %}` after the front matter.